frontend/src/EDR/components/Table.tsx

```diff
@@ -86,9 +87,12 @@ export const EDRTable: React.FC<Props> = ({
             filterConfig={filterConfig}
             setFilterConfig={setFilterConfig}
         />
-        <div>
+        <div className={classNames(
+            "child:snap-y child:snap-mandatory child:overflow-y-scroll ",
+                streamMode ? "child:h-[calc(100vh-102px)]" : "child:h-[calc(100vh-166px)]"
```

Comment:
I added the streamMode size because it makes the header smaller.

I wanted to use a ref, but we cannot really use it so I think this is good enough, given when it is not right it just makes a new scrollbar appears, wdyt ?

Reply:
I completely forgot about `streamMode` thank you for adding. I think this is good.